tools/accuracy_checker/accuracy_checker/metrics/README.md

```diff
@@ -19,6 +19,7 @@ Accuracy Checker supports following set of metrics:
 Supported representations: `ClassificationAnnotation`, `TextClassificationAnnotation`, `ClassificationPrediction`, `ArgMaxClassificationPrediction`.
   * `top_k` - the number of classes with the highest probability, which will be used to decide if prediction is correct.
   * `match` - Batch-oriented binary classification metric. Metric value calculated for each record in batch. Parameter `top_k` ignored in this mode.
+  * `cast_to_int` - in `match` mode predictions rounded to nearest integer before calculating metric value.
```

Comment:
we have postprocessor for this purposes, possibly better to extend it to work with classification data instead manipulation in metric

Reply:
Accuracy metric requires integer prediction values. Roc_auc_score metric assumes that prediction values passed "as is" without casting to int. Cast_to_int postprocessor enabled for classification data but we will loose roc_auc_score.